game-core/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -846,7 +846,7 @@ public boolean getOkToLetUnitsDie(final Collection<Territory> unitsCantFight, fi
     }
     messageAndDialogThreadPool.waitForAll();
     final StringBuilder buf = new StringBuilder("Units in the following territories will die: ");
-    Joiner.on(' ').appendTo(buf, FluentIterable.from(unitsCantFight).transform(unit -> unit.getName()));
+    Joiner.on(' ').appendTo(buf, unitsCantFight.stream().map(unit -> unit.getName()).collect(Collectors.toList()));
```

Comment:
```java
buf.append(unitsCantFight.stream()
  .map(Unit::getName)
  .collect(Collectors.joining(" "));
```

Reply:
fixed in: 59c51cdfc

Reply:
This should work though.

Reply:
Oh, i did not even try it as exactly specified, this was the very first comment I got to @RoiEXLab and I thought it was just formatting. I then moved on. If you would @RoiEXLab please be more specific about the concerns and suggestions you raise so I can be sure I address the right problem.